pkg/auto/bms/config/root.go

```diff
@@ -59,9 +59,17 @@ type Root struct {
 	// If this device supports AirTemperatureHistory then it will be used to seed the weighted average.
 	AutoModeOATemp string `json:"oaTemp,omitempty"`
 
-	OccupancySensors     []string            `json:"occupancySensors,omitempty"`     // Sensors whose occupancy is linked with OccupancyModeTargets On mode.
+	// Settings to control how and when we adjust the OccupancyModeTargets between occupied=on and unoccupied=off.
```

Comment:
I'm not clear on some details of how occupancy sensors interact. Can you answer these questions, and document them?
- I can see a test that uses occupancy sensors and schedules. Is it intended that they be used together?
- What happens if a schedule says occupied but sensors say unoccupied, or vice versa?
- What happens if all occupancy sensors have no data, and there is a schedule?

Reply:
After reading through the comment again and your PR description I think I understand now - the logic is by intersection i.e. if there's a schedule the sensor AND the schedule must report occupancy for occupied mode to be selected. Right?

Reply:
That's correct